packages/web-platform/web-elements/index.css

```
@@ -20,3 +20,4 @@
 @import url("./src/elements/XTextarea/x-textarea.css");
 @import url("./src/elements/XList/x-list.css");
 @import url("./src/elements/XWebView/x-webview.css");
+@import url("./src/elements/XVideo/x-video.css");
```

Stylelint (17.13.0)

[error] Expected "url("./src/elements/XVideo/x-video.css")" to be ""./src/elements/XVideo/x-video.css"" (import-notation)

(import-notation)